### ISSUE TITLE
valueReference in Sample

### ISSUE THREAD

**Issue by philbarker** [BioSchemas/specifications#238]
Please check that valueReference is being used appropriately in the Sample profile. It comes from the GoodRelations ontology which has a [clearer explanation](http://wiki.goodrelations-vocabulary.org/Documentation/Structured_values_and_value_references) than that in schema.org. My paraphrase of that explanation: if you are giving a value for the density of a gas, you need to specify the valueReference to say at what temperature and pressure you made measurement (e.g. Standard Temperature and Pressure)

**Comment by AlasdairGray** [comment]
@mcourtot did you make any progress on this?

**Comment by mcourtot** [comment]
Hi @philbarker and @AlasdairGray, sorry for delayed answer I don't seem to be getting notifications for issues in that repo unless I'm mentioned, so had missed this.
I understand and appreciate the origin of valueReference, but there is no mention of this in the schema.org spec, and I'm wondering if it has undergone a similar modification than this of the sample type - where we wanted to have something more specific than what was ultimately recommended by schema.org. With this in mind I am ok with using valueReference as it currently is in sample as 1. it doesn't seem to contravene any rule 2. I am ok considering that we are providing some context/reference to the term by tagging it with an ontology.

**Comment by simonjupp** [comment]
I agree that valueReference is probably not appropriate here based on the definition from GoodRelations. There's a proposal to move https://schema.org/code into core schema.org and have DefinedTerm as an allowed value. See issue here schemaorg/schemaorg#2243. Thoughts on switching to use code over valueReference?

**Comment by mcourtot** [comment]
can schema.org then update the definition of valueReference?  I'm not sure about code - see comment there wrt identifier.
